### ISSUE TITLE
[Bug Report][3.0.0-alpha.12] ToolTip is show in wrong place and gives an error

### ISSUE THREAD

**Issue by PashaTurok** [vuetifyjs/vuetify#14552]
### Environment
**Vuetify Version:** 3.0.0-alpha.12
**Vue Version:** 3.2.26
**Browsers:** Firefox 95.0
**OS:** Ubuntu undefined

### Steps to reproduce
Move mouse over the button and you will see error in console. ToolTip is shown in wrong position.

### Expected Behavior
ToolTip must be in right position, there mustn't be any errors.

### Actual Behavior
ToolTip is shown in wrong position

### Reproduction Link
<a href="https://codepen.io/pashaturok/pen/wvryZJG" target="_blank">https://codepen.io/pashaturok/pen/wvryZJG</a>

### Other comments
FF gives the following error:

[Vue warn]: Unhandled error during execution of watcher callback 
  at <VOverlay modelValue=true onUpdate:modelValue=fn<onUpdate:modelValue> class="v-tooltip"  ... > 
  at <VTooltip left="" > 
  at <VMain> 
  at <VApp> 
  at <App> runtime-core.esm-bundler.js:6620
Uncaught (in promise) TypeError: ResizeObserver.observe: Argument 1 is not an object.
    connectedPositionStrategy positionStrategies.ts:136
    usePositionStrategies positionStrategies.ts:77
    run reactivity.esm-bundler.js:24
    usePositionStrategies positionStrategies.ts:73
    callWithErrorHandling runtime-core.esm-bundler.js:6737
    callWithAsyncErrorHandling runtime-core.esm-bundler.js:6746
    getter runtime-core.esm-bundler.js:7101
    run reactivity.esm-bundler.js:160
    job runtime-core.esm-bundler.js:7165
    flushPreFlushCbs runtime-core.esm-bundler.js:6910
    flushPreFlushCbs runtime-core.esm-bundler.js:6916
    updateComponentPreRender runtime-core.esm-bundler.js:4470
    componentUpdateFn runtime-core.esm-bundler.js:4382
    run reactivity.esm-bundler.js:160
....

**Comment by Michael-372** [comment]
I'm having the same issue and did some digging into the Vuetify code to try and find the culprit. It appears that the activator is not being properly pushed into the V-Overlay component (v-tooltip looks like a wrapper for v-overlay) when using the activator slot for v-tooltip. 

I noticed this because the "activatorEl" was undefined inside the V-Overlay props using the Vue Dev Tools.

While digging around the [source code](https://github.com/vuetifyjs/vuetify/blob/2bf113caad20a7654aed8603fb7c6c2b2cfebfb5/packages/vuetify/src/components/VOverlay/useActivator.tsx#L188), I figured out that the following syntax will work as a workaround:

```   
<v-tooltip anchor="left" activator="#myActivator">
  <span>Hello world</span>
</v-tooltip>
<v-btn
  icon="mdi-play"
  @click="doSomething()"
  id="myActivator"
/>
```
